dashboard/config/scripts/levels/AI Chat Test Level.level

```diff
@@ -1,12 +1,14 @@
 <Aichat>
   <config><![CDATA[{
-  "properties": {"encrypted":"false","contained_level_names":null},
-  "game_id": 71,
   "published": true,
-  "created_at": "2023-04-05T08:57:39.000Z",
+  "game_id": 71,
+  "created_at": "2023-08-04T05:25:28.000Z",
   "level_num": "custom",
   "user_id": 4,
-  "audit_log": "[{\"changed_at\":\"2023-04-05 09:03:03 +0000\",\"changed\":[\"published\"],\"changed_by_id\":4,\"changed_by_email\":\"brendan+levelbuilder@code.org\"}]"
+  "properties": {
+    "encrypted": "false",
+    "long_instructions": "## Sample Instructions\r\n\r\nThis is a chat bot which allows you to talk to historical figures.\r\n\r\n### Do This\r\n\r\nType in a question and press the `Submit`(#7435b2) button. What can you learn about George Washington?"
```

Comment:
When I looked at the edit levelbuilder page, I noticed that the text 'Submit' does not show up in the preview even though it does on the level page.

![Screen Shot 2023-08-17 at 10 17 40 AM](https://github.com/code-dot-org/code-dot-org/assets/107423305/281c5b6a-c48a-484a-ac86-72f6e7bfe216)

Reply:
Yeah I think this just might be a difference between the markdown preview here vs on the level...looks like the Submit text is there, just very hard to see due to the text color against the purple background.